googleapiclient/errors.py

```diff
@@ -48,7 +48,11 @@ def _get_reason(self):
         """Calculate the reason for the error from the response content."""
         reason = self.resp.reason
         try:
-            data = json.loads(self.content.decode("utf-8"))
+            try:
+                data = json.loads(self.content.decode("utf-8"))
+            except:
```

Comment:
I'd prefer the specific exception be caught instead. It looks like it's `json.JSONDecodeError` in this case. https://docs.python.org/3/library/json.html#json.JSONDecodeError

Reply:
Done